rust/hyperlane-base/src/settings/base.rs

```diff
@@ -49,7 +49,7 @@ pub struct Settings {
     /// Configuration for contracts on each chain
     pub chains: HashMap<String, ChainConf>,
     /// Port to listen for prometheus scrape requests
-    pub metrics: Option<u16>,
+    pub metrics: u16,
```

Comment:
nit: If it's easy, suggest metrics_port

Reply:
This one is easy, changing the name in the actual config is not.